clock_test.go

```diff
@@ -4,14 +4,18 @@
 package testing_test
 
 import (
-	gc "gopkg.in/check.v1"
 	"sync"
 	"time"
 
+	gc "gopkg.in/check.v1"
+
 	"github.com/juju/testing"
```

Comment:
As discussed in person at the sprint, this conforms with how block are separated in juju/cmd and other libraries outside of juju/juju. Also, it is how goimports auto formats imports. :)